modules/applications/examples/ui-builder/production/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+# Used for running a bonita env for production
+services:
+  bonita-ui-builder:
+    image: <my-uib-image-name>:<version>
+    container_name: bonita-ui-builder
+    environment:
+      BONITA_DEV_MODE: false
```

Comment:
This is not needed

Reply:
This is in case the build was done without the parameter (as default value is true)

Reply:
@jeromecambon Indeed, this is mandatory to `false` to build the productionb image :wink: